README.md

```diff
@@ -86,46 +87,87 @@ Ensure you have the following installed:
 
 Follow these steps to set up and run the project using Docker.
 
-1. **Build and Start the Container**
+1. **Build and Start the Containers**
 
     ```bash
-    # build the container
+    # pull the supporting containers
+    make compose-pull
+
+    # build the main application image
     make docker-build
-    # start the container
-    make docker-start
+
+    # start the containers
+    make compose-up
     ```
 
 2. **Migrations and Superuser Creation**
 
     If this is your first time setting up the project, you’ll need to run migrations to set up the database schema and create an administrative user.
-    Note: `dj` is an alias for `django-admin`
 
     ```bash
     # ssh into the web container
     make docker-shell
 
     # Run database migrations
-    dj migrate
+    make migrate
 
     # Create a superuser for accessing the admin interface
-    dj createsuperuser
+    make createsuperuser
     ```
 
 3. **Start Django Inside the Container**
 
     Once the containers are running, you need to manually start Django from within the web container. This allows for running both the Django server and any additional background services (e.g., schedulers).
 
+> [!WARNING]  
+> The `honcho` command will pick up your local mounted `.env` file when running via `docker-compose`. Ensure that you comment out any local variables which might cause clashes in the container context as they will take precedence when running `honcho start`.
```

Comment:
This breaks the indentation and doesn't escape the markdown. I suggest just using a warning emoji, as it doesn't look like you can use indented panels.
<img width="129" alt="image" src="https://github.com/user-attachments/assets/68241581-41b1-403c-92b8-7c0568a92e41">